docs/index.md

```diff
@@ -113,7 +113,7 @@ Versions in Linux distributions:
 * [1.2.21](Release-Notes-1.2.21)
 * [1.2.20](Release-Notes-1.2.20)
 * [1.2.19](Release-Notes-1.2.19)
-* [1.2.18](Release-Notes-1.2.18)
+* [1.2.18](Release-Notes-1.2.18) - Unconditional setup resolver config and add MPS personalities
```

Comment:
AuthorDate: Fri Jun 10 11:09:41 2016 +0200
Please specify date (2016-06-10)

Reply:
fixed --nocheck option, added Mageia configs, added Custom config, /etc/machine-id kept, use DNF for F24+